plugins/core/src/main/kotlin/de/fayard/refreshVersions/core/internal/versions/VersionsPropertiesReading.kt

```diff
@@ -20,7 +20,7 @@ internal fun VersionsPropertiesModel.Companion.readFromText(
     )
 } catch (e: IllegalArgumentException) {
     throw IllegalStateException(e)
-}
+}.apply { this.sections.forEach(::println) }
```

Comment:
I'm pretty sure this doesn't do what you intended at all, I think this change should be reverted.

@jmfayard what did you want to do?

Reply:
that's correct, I already reverted it in 12227b78a167d87fc4d1c7057901d67803c8923c